### DISCUSSION TITLE
Keyboard shortcuts don't work unless window is out of 'Full Screen' mode?

### DISCUSSION THREAD

**Discussion by stevecondylios** [rxhanson/Rectangle#1514] [General]
This might be a bug in my understanding than in the software itself. 

For the first 20 minutes trying to use Rectangle, I kept pressing <kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>left</kbd> to make a full screen<sup>1</sup> window go to the 'left half' of the screen. But I couldn't get it to work. I tried enabling all the Accessibility options in Settings but they were enabled, and tried a few things I found in GitHub issues.

Is this supposed to work? Or do Rectangle shortcuts like <kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>left</kbd> ***only*** work on windows that are **not** in 'full screen'? - i.e. I notice <kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>left</kbd> *does* work when the window is any other size, including when it's *almost* full screen (achieved by double clicking a window's title bar). But this seems significantly less useful (since 'full screen' is extremely common).

The best approach I found to go from 'full screen' to 'window to the left', is:

1. press <kbd>ctrl</kbd>+<kbd>command</kbd>+<kbd>f</kbd> to get out of 'full screen', then
2. <kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>left</kbd> for 'Left half'

Hopefully I'm mistaken and there's a single shortcut (or some setting I must change to allow this).

Notes: 

- 'Left half' (<kbd>ctrl</kbd>+<kbd>opt</kbd>+<kbd>left</kbd>) is just an example, I presume the same thing happens with all other rectangle shortcuts.
- macOS Sonoma 14.4.1 (23E224)

<sup>1</sup> 'Full screen' mode, as achieved by pressing the green button on top left of a window

**Comment by rxhanson** [comment]
Thanks for the feedback!

There is not a single shortcut or a setting that allows this, and I understand how this would be a confusing scenario. I don't have the bandwidth/desire to add this in myself at the moment, but see no reason to turn down a good pull request for it. 

I almost never use full screen, and I use Rectangle to maximize windows.
